yt_dlp/extractor/likee.py

```diff
@@ -0,0 +1,192 @@
+            'download_count': int,
+            'timestamp': 1651506835,
+            'upload_date': '20220502',
+            'duration': 60024,
+            'play_count': int,
+            'artist': 'Vương Phước Nhi',
+            'uploader_id': '649222262',
+            'view_count': int,
+        },
+    }, {
+        'url': 'https://likee.video/@fernanda_rivasg/video/6932224568407629502',
+        'info_dict': {
+            'id': '6932224568407629502',
+            'ext': 'mp4',
+            'title': 'Un trend viejito🔥 #LIKEE #Ferlovers #trend ',
+            'description': 'md5:c42b903a72a99d6d8b73e3d1126fbcef',
+            'thumbnail': r're:^https?://.+\.jpg',
+            'comment_count': int,
+            'duration': 9684,
+            'uploader_id': 'fernanda_rivasg',
+            'view_count': int,
+            'play_count': int,
+            'artist': 'La Cami La✨',
+            'download_count': int,
+            'like_count': int,
+            'uploader': 'Fernanda Rivas🎶',
+            'timestamp': 1614034308,
+            'upload_date': '20210222',
+        },
+    }, {
+        'url': 'https://likee.video/v/k6QcOp',
+        'info_dict': {
+            'id': 'k6QcOp',
+            'ext': 'mp4',
+            'title': '#AguaChallenge t&uacute; ya lo intentaste?😱🤩',
+            'description': 'md5:b0cc462689d4ff2b624daa4dba7640d9',
+            'thumbnail': r're:^https?://.+\.jpg',
+            'comment_count': int,
+            'duration': 18014,
+            'play_count': int,
+            'view_count': int,
+            'timestamp': 1611694774,
+            'like_count': int,
+            'uploader': 'Fernanda Rivas🎶',
+            'uploader_id': 'fernanda_rivasg',
+            'download_count': int,
+            'artist': 'ʟᴇʀɪᴋ_ᴜɴɪᴄᴏʀɴ♡︎',
+            'upload_date': '20210126',
+        },
+    }, {
+        'url': 'https://www.likee.video/@649222262/video/7093167848050058862',
+        'only_matching': True,
+    }, {
+        'url': 'https://www.likee.video/v/k6QcOp',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        info = self._parse_json(
+            self._search_regex(r'window\.data\s=\s({.+?});', webpage, 'video info'),
+            video_id, transform_source=js_to_json)
+        video_url = traverse_obj(info, 'video_url', ('originVideoInfo', 'video_url'))
+        if not video_url:
+            self.raise_no_formats('Video was deleted', expected=True)
+        formats = [{
+            'format_id': 'mp4-with-watermark',
+            'url': video_url,
+            'height': info.get('video_height'),
+            'width': info.get('video_width'),
+        }, {
+            'format_id': 'mp4-without-watermark',
+            'url': video_url.replace('_4', ''),
+            'height': info.get('video_height'),
+            'width': info.get('video_width'),
+        }]
```

Comment:
```suggestion
            'format_id': 'mp4-without-watermark',
            'url': video_url.replace('_4', ''),
            'height': info.get('video_height'),
            'width': info.get('video_width'),
            'quality': 1,
        }]
```
since they have same width/height, _sort_formats wouldn't know how to sort them properly